packages/hooks/src/test/index.test.js

```diff
@@ -64,6 +64,14 @@ function action_c() {
 	window.actionValue += 'c';
 }
 
+function handle_hook_added ( ...args ) {
+	window.hookAdded = { ...args };
```

Comment:
I'm sure it could be tested using local scope with `let hookAdded, hookRemoved` defined outside of the functions that use them.

Reply:
Another option is described below: https://github.com/WordPress/packages/pull/52/files#r157469346.

Reply:
👍 sounds good if you want to change that